### DISCUSSION TITLE
[FEATURE-REQUEST] Botón Acciones Primarias y Secundarias

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#10] [3. Ideas & Feature Requests] [answered]
Hola, 

Tengo una consulta sobre el botón para crear canales o tasks o levantar formularios:
Me di cuenta que sí presiono por error el botón de crear canal o tareas y lo quiero cerrar (no me es intuitivo) debo presionar encima del fondo azul. 

Mi pregunta es: ¿tendrá un botón X para cerrar el cuadro? o ¿Está configurado de tal manera que para cancelar se debe presionar en dicho fondo?. 

![image](https://user-images.githubusercontent.com/3951895/113207360-e7d82400-9246-11eb-836c-88e5dc44f769.png)

Gracias

**Comment by acedward** [comment, marked as answer]
Hola,

Sí el cierre está a propósito sobre cualquier sección externa (es como normalmente funciona este tipo de botón)
Es problemático que no se cierre aprentando sobre cualquier parte o apretando escape en el teclado.
